### ISSUE TITLE
v0.36.0-rc2 Dashboards with text box cards fail to load

### ISSUE THREAD

**Issue by lukeman** [metabase/metabase#12873]
**Describe the bug**
Testing a local copy of my prod instance and many of my existing dashboards fail to load with `undefined is not an object (evaluating 'e.type')`. Checking each card individually seemed fine, so I was able to isolate that you get this error on any dashboard (whether new or before the upgrade) that contains a text box.

**Logs**
Nothing relevant in the app logs. Here's the full minified console traceback:

```
TypeError: Cannot read property 'type' of undefined
    at Function.value (app-main.bundle.js?99a5979f407148c10717:5)
    at e.value (app-main.bundle.js?99a5979f407148c10717:5)
    at app-main.bundle.js?99a5979f407148c10717:5
    at i (app-main.bundle.js?99a5979f407148c10717:5)
    at e.n.value (app-main.bundle.js?99a5979f407148c10717:5)
    at app-main.bundle.js?99a5979f407148c10717:5
    at Array.map (<anonymous>)
    at app-main.bundle.js?99a5979f407148c10717:5
    at app-main.bundle.js?99a5979f407148c10717:5
    at dispatch (vendor.bundle.js?99a5979f407148c10717:1)
```

**To Reproduce**
1. Create a new dashboard
2. Switch to edit mode
3. Add a text box
4. Save & Reload
5. 💥 

**Expected behavior**
Dashboards with text boxes should load.

**Screenshots**
Creating a new dashboard works fine. Saves fine.
<img width="1001" alt="Screen Shot 2020-07-08 at 5 08 02 PM" src="https://user-images.githubusercontent.com/13008/86970857-215b8300-c13e-11ea-8bf1-eb903b5bd849.png">

Reloading the page gives the following error.
<img width="1001" alt="Screen Shot 2020-07-08 at 5 08 08 PM" src="https://user-images.githubusercontent.com/13008/86970867-2587a080-c13e-11ea-8d47-73e5b86ec998.png">

**Severity**
I'd assume it's a blocker for a release, since there's seemingly no way to fix affected dashboards other than to roll back.

**Metabase Diagnostic Info**

```json
{
  "browser-info": {
    "language": "en",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.1 Safari/605.1.15",
    "vendor": "Apple Computer, Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "Java(TM) SE Runtime Environment",
    "java.runtime.version": "1.8.0_191-b12",
    "java.vendor": "Oracle Corporation",
    "java.vendor.url": "http://java.oracle.com/",
    "java.version": "1.8.0_191",
    "java.vm.name": "Java HotSpot(TM) 64-Bit Server VM",
    "java.vm.version": "25.191-b12",
    "os.name": "Mac OS X",
    "os.version": "10.15.5",
    "user.language": "en",
    "user.timezone": "America/New_York"
  },
  "metabase-info": {
    "databases": [
      "bigquery",
      "mysql",
      "postgres"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "9.5.21"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.8"
      }
    },
    "run-mode": "prod",
    "version": {
      "tag": "v0.36.0-rc2",
      "date": "2020-07-08",
      "branch": "remove-equiv-method-implementations",
      "hash": "5956b4d"
    },
    "settings": {
      "report-timezone": "US/Pacific"
    }
  }
}
```

**Comment by BarreiroLindoF** [comment]
I had the same problem. The only quickfix I found was to remove the text box cards from the dashboard in the database.

If you want to do it, you have to connect to your metabase database, execute the following SQL query : 

`DELETE FROM report_dashboard WHERE dashboard_id = 1000 AND card_id IS NULL;`

Replace 1000 by your dashboard_id. To find the id, you can go to your metabase web application, navigate to your dashboard and the link should look like : 

https://metabase.company.com/dashboard/1000

### PLEASE BE CAREFUL NOT TO LOSE ANY DATA ! USE AT OWN RISK

**Comment by paulrosenzweig** [comment]
Fixed in #11358 (it wasn't the main point of that PR)
